### ISSUE TITLE
[R-Forge #1231] Compile j to byte code?

### ISSUE THREAD

**Issue by arunsrinivasan** [Rdatatable/data.table#665]
Submitted by: Matt Dowle; Assigned to: Nobody; [R-Forge link](https://r-forge.r-project.org/tracker/index.php?func=detail&aid=1231&group_id=240&atid=978)

Have made some first steps on this, looks hard.  See Luke T's webpages on the subject from a long time ago.

**Comment by jangorecki** [comment]
I don't think it make sense to try to handle that inside data.table. It is pretty complex tasks to disassemble expressions and handle compilation properly. I would rather prefer to have that part outsourced to another package, and just ensure it plays well inside `j`. Voting to close.
